sat-plugin/src/main/java/org/openhab/tools/analysis/tools/SpotBugsChecker.java

```diff
@@ -82,7 +82,7 @@
     /**
      * The version of the spotbugs-maven-plugin that will be used
      */
-    @Parameter(property = "maven.spotbugs.version", defaultValue = "3.1.5")
+    @Parameter(property = "maven.spotbugs.version", defaultValue = "3.1.7")
     private String spotbugsMavenPluginVersion;
```

Comment:
There's a Spotbugs Maven plugin and Spotbugs itself. 

The most recent released version of: 
* Spotbugs Maven plugin is 3.1.6
* Spotbugs is 3.1.7 (the field below, which also needs to be updated.

Reply:
This was already out-of-sync before, good that you noticed 👍 

Let me also update the documentation